Examples/SpeechRecognition/SpeechRecognition/SpeechClient/Live.swift

```diff
@@ -24,24 +24,17 @@ extension SpeechClient {
       },
       startTask: { request in
         Effect.run { subscriber in
-          let speechRecognizer = SFSpeechRecognizer(locale: Locale(identifier: "en-US"))!
-          let cancellable = AnyCancellable {
```

Comment:
this cancellable was way to high up, need to move down below after we create audio engine and recognition task.